src/constructs/iam/policies/parameter-store-read.ts

```diff
@@ -3,19 +3,19 @@ import type { GuStack } from "../../core";
 import { AppIdentity } from "../../core/identity";
 import { GuPolicy } from "./base-policy";
 
+export function guParameterStoreReadPolicyStatement(scope: GuStack, app: string): PolicyStatement {
+  return new PolicyStatement({
+    effect: Effect.ALLOW,
+    actions: ["ssm:GetParametersByPath"],
+    resources: [`arn:aws:ssm:${scope.region}:${scope.account}:parameter/${scope.stage}/${scope.stack}/${app}`],
+  });
+}
```

Comment:
The library currently defines constructs as classes, can we convert this to a class for consistency?

Reply:
Makes sense to be consistent. Can you have another look?